data/scenarios/Tutorials/give.yaml

```diff
@@ -47,17 +74,27 @@ robots:
       - [10, scanner]
       - [10, treads]
       - [10, solar panel]
+  - name: replacer
+    system: true
+    program: |
+      def replace: Cmd Unit =
+          i <- scan down;
+          watch down; wait 2024; wait 32;
```

Comment:
This is an example where I think it's helpful to have a version of `watch` which *only* pays attention to entity changes, not robots entering the cell --- because we have nothing useful to do when a robot shows up, only when the entity disappears.

Reply:
I agree that this use only cares about entity changes, but it has to handle errors anyway.

It is generally wrong to assume that if watching robot was woken up, then the cell now (doesn't) contain an entity. Other robots could have gone before it.

That "something changed in the cell at least once" is all the robot knows.

But it could have also been woken up at the end of wait time, as I did not implement infinity yet.

Reply:
True, all good points.